Icemoon Ruins (+2)

MrFagetti opened the pull request:
## About The Pull Request

Adds two new Icemoon locales

## Why It's Good For The Game

Fun to explore and loot

## Changelog
:cl:
add: Icemoon ruin - Gas Station
add: Icemoon ruin - Military Ship wreckage
/:cl:

coldud13 commented:
lotta food but the servo map looks alright
what kinda locker is that in the syndie ship tho?

MrFagetti commented:
Fixed the locker, changed it to a secure case, so you gotta kinda work for the special stuff. Otherwise, the ruins pretty much just O2 tanks, ya know? Shame that nobody wants to play Snaxi to look at them, and Fridge is in a weird spot still.
